Quadriga/src/main/java/edu/asu/spring/quadriga/web/LoginController.java

```diff
@@ -25,19 +30,27 @@
 	IUserManager userManager;
 	IUser user;
 
+    @Autowired 
+	private IRetrieveProjectManager projectManager;
+    
 	/**
 	 * A valid authenticated user is redirected to the home page.
 	 * 
 	 * @return 		Returned to the home page of Quadriga.
+	 * @throws QuadrigaStorageException 
 	 */
 	@NoAuthorizationCheck
 	@RequestMapping(value = "auth/welcome", method = RequestMethod.GET)
 	public String validUserHandle(ModelMap model, Principal principal,
-			Authentication authentication) {
+			Authentication authentication) throws QuadrigaStorageException {
 
 		// Get the LDAP-authenticated userid
 		String sUserId = principal.getName();		
 		model.addAttribute("username", sUserId);
+
+        List<IProject> recentProjects = new ArrayList<IProject>();
+        recentProjects = projectManager.getRecentProjectList(sUserId);
+		model.addAttribute("projects", recentProjects);
 		
 		return "auth/home";

```

Comment:
I think, this should just redirect to "auth/home". Would you please remove line 48 to 53 and change the return value to "redirect:auth/home"